app/src/bin/crate.in.sh

```diff
@@ -80,4 +80,7 @@ JAVA_OPTS="$JAVA_OPTS -XX:+DisableExplicitGC"
 JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"
 
 # Use our provided JNA always versus the system one
-JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true"
\ No newline at end of file
+JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true"
```

Comment:
can be written as

`JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true -Dlog4j.shutdownHookEnabled=false"`

also, why don't we provide the other two vm options?
https://github.com/elastic/elasticsearch/blob/ee802ad63c0f21d697a5095dd05dc6f94626ee4d/distribution/src/main/resources/config/jvm.options#L73-L76